bin/docsify

```diff
@@ -45,10 +45,18 @@ var argv = require('yargs')
     handler: (argv) => run.init(argv.path, argv.local, argv.theme)
   })
   .command({
-    command: 'serve <path> [--port 3000]',
+    command: 'serve <path> [--disable-auto-open false] [--port 3000]',
```

Comment:
I think we should use `--open`.

- `disable-auto-open` is too long
- The default auto open browser is not friendly